src/js/components/FormField/FormField.js

```diff
@@ -171,14 +171,29 @@ class FormFieldContent extends Component {
       if (abut) {
         // marginBottom is set to overlap adjacent fields
         let marginBottom = '-1px';
+        let marginTop = '0px';
+        let marginRight = '0px';
+        let marginLeft = '0px';
         if (border.size) {
           marginBottom = `-${parseMetricToNum(
             theme.global.borderSize[border.size],
           )}px`;
+          marginTop = `-${parseMetricToNum(
+            theme.global.borderSize[border.size],
```

Comment:
maybe we can avoid doing this calculation 4 times

Reply:
Okay, will fix this.

Reply:
@oorestisime something in line of 
```
static defaultProps = {
 margin: {
  top: 'top',
  bottom: 'bottom',
  right: 'right',
  left: 'left',
 }
}
```

will work ?